### ISSUE TITLE
C# -> VB: Event keyword added to Field Declaration with Lambda Expression

### ISSUE THREAD

**Issue by RussellSmith2** [icsharpcode/CodeConverter#490]
### Input code
```csharp
using System;

namespace ICSharpCode.CodeConverter.Util
{
	static class CSharpClass
	{
		private static readonly Func<object, bool> s_notNullTest = x => x != null;
	}
}
```

### Erroneous output
```vbnet
Imports System

Namespace ICSharpCode.CodeConverter.Util
    Friend Module CSharpClass
        Private ReadOnly Event s_notNullTest As Func(Of Object, Boolean) = Function(x) x IsNot Nothing
    End Module
End Namespace
```

### Expected output
```vbnet
Imports System

Namespace ICSharpCode.CodeConverter.Util
    Friend Module CSharpClass
		Private ReadOnly s_notNullTest As Func(Of Object, Boolean) = Function(x) x IsNot Nothing
    End Module
End Namespace
```

### Details
An Event keyword is added when converting a Field Declaration with Lambda Expression.

* **Event keyword is only added when "using System;" is part of the original C# text.**

* Version in use: Current compiled repository

**Comment by GrahamTheCoder** [comment]
Related to issue being fixed in https://github.com/icsharpcode/CodeConverter/pull/472

**Comment by GrahamTheCoder** [comment]
Was fixed in #472
